prac_05/word_occurrences.py

```diff
@@ -0,0 +1,17 @@
+words_to_count = {}
+
+user_string = input("Please enter a string: ")
+words = user_string.split()
+for word in words:
+    if word in words_to_count:
+        words_to_count[word] += 1
+    else:
+        words_to_count[word] = 1
+
+words = list(words_to_count.keys())
+words.sort()
+
+longest_word = max((len(word) for word in words))
```

Comment:
I like this code. Smart.

Reply:
I was actually struggling with this code so I don't know if you are joking or not, the code does not work as intended and i can't get the right formatting at the end.